user-guide/Troubleshooting/Known_issues/KI_Cassandra_backups_not_working.md

```diff
@@ -0,0 +1,63 @@
+---
+uid: KI_Cassandra_backups_not_working
+---
+
+# Cassandra backups no longer working
+
+## Affected versions
+
+DataMiner 10.3.9 CU0.
+
+## Cause
+
+DataMiner 10.3.9 introduced additional binding redirects in the CassandraBackup.exe.config file. SLDataGateway would try to load these references, but as these were not available in `C:\Skyline DataMiner\Tools`, this caused exceptions.
+
+## Workaround
```

Comment:
Is a DataMiner restart needed for this? Also, does this need to be done on every DMA in a cluster?